synthesizer/src/vm/finalize.rs

```diff
@@ -2338,6 +2379,7 @@ finalize compute:
         // Check that all mappings in `credits.aleo` are equal across the two VMs.
         let program_id = ProgramID::from_str("credits.aleo").unwrap();
         let committee_mapping_name = Identifier::from_str("committee").unwrap();
+        let commission_mapping_name = Identifier::from_str("commission").unwrap();
```

Comment:
future improvement idea: while there might be many of them, it might be a good idea to have such literals `const`-ified, so we don't need to worry about any misspelling